story/management/commands/fetch_tumblr_stories.py

```diff
@@ -0,0 +1,52 @@
+from typing import Tuple
+from urllib.parse import urlparse
+from urllib.request import urlretrieve
+
+from django.core.files import File
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from story.models import Story
+
+from core.tumblr_client import request_latest_stories
+
+
+def download_image(url: str) -> Tuple[str, File]:
+    image_name = urlparse(url).path.split("/")[-1]
+    content = urlretrieve(url)
+    return (image_name, File(open(content[0], "rb")))
```

Comment:
You could drop the outer braces from this & it'll still return a `tuple`.

```suggestion
    return image_name, File(open(content[0], "rb"))
```